metrics/prometheus/collector.go

```diff
@@ -50,13 +50,16 @@ func newCollector() *collector {
 // Add adds the metric i to the collector. This method returns an error if the
 // metric type is not supported/known.
 func (c *collector) Add(name string, i any) error {
+	fmt.Println(name, i)
```

Comment:
Is this intentional or a leftover from testing? I assume it's the latter
```suggestion
```

Reply:
Whoops, I did remembed looking for it and not finding. Thought I already removed it